pgd_core/models.py

```diff
@@ -325,6 +325,8 @@ class Residue(models.Model):
     bm              = models.FloatField()
     bs              = models.FloatField()
     bg              = models.FloatField(null=True)
+    occm            = models.FloatField(null=True)
```

Comment:
These changes satisfy #17553 (adding to the database).  They ask for "single precision floating point", you've given them FloatField, we're good.